### DISCUSSION TITLE
Customer Management Framework Error on Customer List

### DISCUSSION THREAD

**Discussion by AlephLau** [pimcore/pimcore#14043] [Q&A / Support] [answered]
Could you help me with this problem?
I just installed CMF and this problem appears, it should be noted that the object_1 table is empty.
Thank you 

![Screenshot from 2023-01-11 10-03-53](https://user-images.githubusercontent.com/70108013/211869856-995e3152-61f3-40cb-88e3-b91a0f4cfea6.png)

**Comment by mattamon** [comment, marked as answer]
Hey @AlephCode,

these seems odd, like the class definitions did not get installed correctly. 

Can you check your classes if the `filterSortOrder` is under the options?
It should look like this:
![image](https://user-images.githubusercontent.com/38959016/212070477-0d957102-41d4-452f-8676-b8a05a736744.png)

**Comment by AlephLau** [comment]
Thank you @mattamon 
You are right, what can you do to correct it? Do I have to reinstall CFM waiting for it to create these new objects?
Would it be wise to export this class from the pimcore demo and import it into my project?
![image](https://user-images.githubusercontent.com/70108013/212122750-6a7171be-5c71-4c1d-882d-43bc372d335e.png)
![image](https://user-images.githubusercontent.com/70108013/212123073-51c50109-a8a7-4b1f-9f4d-a601887bd7b6.png)

**Reply by mattamon** [reply]
If there are more properties missing, you can export it from demo and import it in your project for sure! 
Otherwise just add it manually. No harm in that :) 

But maybe check your whole installation, seems like something went wrong tbh.

**Reply by AlephLau** [reply]
I exported the demo pimcore class and imported it into my project and it works! Thank you so much! :D

But now I'm a bit worried because maybe something else got installed wrong.

But my colleagues and I have followed the documentation step by step and this problem appeared to all of us.

**Reply by mattamon** [reply]
This is strange. Which version of pimcore are you using and which version of the CMF did you try to install? :)

**Reply by AlephLau** [reply]
Hi @mattamon 
I used the command composer require pimcore/customer-management-framework-bundle and it automatically installed version 3.3.6
Perhaps something has been installed wrong since I have found some errors such as when I search in the search engine of the data object folder of my customers it does not return any results.
